cli/config/configfile/file.go

```diff
@@ -44,6 +44,7 @@ type ConfigFile struct {
 	NodesFormat          string                      `json:"nodesFormat,omitempty"`
 	PruneFilters         []string                    `json:"pruneFilters,omitempty"`
 	Proxies              map[string]ProxyConfig      `json:"proxies,omitempty"`
+	ExperimentalCli      string                      `json:"experimentalCli,omitempty"`
```

Comment:
`CLI` is an acronym, so this should be `ExperimentalCLI` (and `experimentalCLI` for JSON)

Given that this configuration file is for the CLI; is `CLI` a bit redundant? Perhaps just `Experimental`?